css-fonts-5/Overview.bs

```diff
@@ -93,13 +93,228 @@ Issue(4044):
 <h3 id="font-size-adjust-prop">
 Relative sizing: the 'font-size-adjust' property</h3>
 
-The 'font-size-adjust' property is applied after the 'size-adjust' descriptor.
+	<pre class="propdef">
+	Name: font-size-adjust
+	Value: none | [ ex | cap | ch | ic ]? [ from-font | <<number>> ]
+	Initial: none
+	Applies to: all elements and text
+	Inherited: yes
+	Percentages: N/A
+	Computed value: the keyword ''font-size-adjust/none'', or a map of the specified keywords and their associated <<number>>
```

Comment:
> a map of…

Is there precedent for the computed value being a map? How is that different from just a tupple of the keyword and the associated number?

Reply:
hmm, I got this wrong entirely actually. There's only ever one keyword-value pair.

Reply:
Fixed.